megamek/src/megamek/client/ui/swing/boardview/StepSprite.java

```diff
@@ -465,38 +465,32 @@ private void drawMovementCost(MoveStep step, boolean isLastStep,
 
         // If the step is using a road bonus, mark it.
         if (step.isOnlyPavement() && e.isEligibleForPavementBonus()) {
-            costStringBuf.append("+");
+            costStringBuf.append('+');
         }
 
         // Show WiGE descent bonus
         for (int i = 0; i < step.getWiGEBonus(); i++) {
-            costStringBuf.append("+");
+            costStringBuf.append('+');
         }
 
         // If the step is dangerous, mark it.
         if (step.isDanger()) {
-            costStringBuf.append("*");
+            costStringBuf.append('*');
         }
 
         // If the step is past danger, mark that.
         if (step.isPastDanger()) {
-            costStringBuf.insert(0, "(");
-            costStringBuf.append(")");
+            costStringBuf.insert(0, '(');
+            costStringBuf.append(')');
         }
 
         EntityMovementType moveType = step.getMovementType(isLastStep);
-        if ((moveType == EntityMovementType.MOVE_VTOL_WALK)
-                || (moveType == EntityMovementType.MOVE_VTOL_RUN)
-                || (moveType == EntityMovementType.MOVE_VTOL_SPRINT)
-                || (moveType == EntityMovementType.MOVE_SUBMARINE_WALK)
-                || (moveType == EntityMovementType.MOVE_SUBMARINE_RUN)) {
-            costStringBuf.append("{").append(step.getElevation())
-                    .append("}");
+        if ((moveType == EntityMovementType.MOVE_VTOL_WALK) || (moveType == EntityMovementType.MOVE_VTOL_RUN) || (moveType == EntityMovementType.MOVE_VTOL_SPRINT) || (moveType == EntityMovementType.MOVE_SUBMARINE_WALK) || (moveType == EntityMovementType.MOVE_SUBMARINE_RUN)) {
```

Comment:
Really not convinced this is easier to read.

Reply:
IDE did that to me. I will revert.